src/coreclr/jit/compiler.hpp

```diff
@@ -5145,37 +5145,18 @@ BasicBlockVisit FlowGraphNaturalLoop::VisitLoopBlocks(TFunc func)
 template <typename TFunc>
 BasicBlockVisit FlowGraphNaturalLoop::VisitLoopBlocksLexical(TFunc func)
```

Comment:
We should likely just remove it. The usages are just for duplicating loop blocks, and those probably don't need to be duplicated lexically anymore.

No need to make the change here if you don't want to, I can do it separately.

Reply:
Got it, I saw #110020 is introducing new call sites for it, so I decided to keep it around to avoid conflicts with that PR. I can remove it in one go once that PR is in.